.env

```diff
@@ -20,3 +20,7 @@ SITE_NAME=''
 USER_INFO_COOKIE_NAME=''
 APP_ID=''
 MFE_CONFIG_API_URL=''
+ALGOLIA_APP_ID=''
+ALGOLIA_JOBS_INDEX_NAME=''
+ALGOLIA_PRODUCT_INDEX_NAME=''
+ALGOLIA_SEARCH_API_KEY=''
```

Comment:
I don't see these constants being added in any of the tickets you mentioned, is this just another missed change?

Reply:
nevermind just looked at https://github.com/edx/edx-internal/pull/8737/files